07_errors/willshen8/storer_test.go

```diff
@@ -0,0 +1,132 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+)
+
+type StorerSuite struct {
+	suite.Suite
+	storer        Storer
+	Puppy1        Puppy
+	Puppy2        Puppy
+	Puppy3        Puppy
+	Puppy4        Puppy
+	Puppy5        Puppy
+	Puppy6        Puppy
+	Puppy7        Puppy
+	InvalidPuppy  Puppy
+	ErrorCreateID uint32
+	ErrorCreate   string
+	ErrorRead     string
+	ErrorUpdate   string
+	ErrorDelete   string
+}
+
+func (s *StorerSuite) SetupTest() {
+	s.Puppy1 = Puppy{1, "Jack Russell Terrier", "White and Brown", "1500"}
```

Comment:
Move these to the variables at the top of the file.

Reply:
Done!